src/useScreenOrientation/useScreenOrientation.ts

```diff
@@ -9,7 +9,7 @@ export type ScreenOrientation = 'portrait' | 'landscape';
  * hook uses CSS3 `orientation` media-query to check screen orientation.
  */
 export function useScreenOrientation(): ScreenOrientation | undefined {
-  const matches = useMediaQuery('(orientation: portrait)');
+  const matches = useMediaQuery('(orientation: portrait)', { initializeWithValue: false });
```

Comment:
Maybe we should also add an argument to this hook to yield correct value on init

Reply:
Good point. Did not even think about this hook too much 😄 I'll work on this tomorrow.

Reply:
Done.